src/pages/_app.tsx

```diff
@@ -16,6 +16,10 @@ function MyApp({ Component, pageProps }: ComponentWithPageLayout) {
   // From documentation on using Posthog with Next.js: https://posthog.com/docs/integrate/third-party/next-js
   const router = useRouter();
 
+  useEffect(() => {
+    import('preline')
```

Comment:
So there seems to be an issue with the deployment. Seems like TypeScript can't find type definitions for this import for some reason. Could you look into that and see if you can resolve it please?

Reply:
> So there seems to be an issue with the deployment. Seems like TypeScript can't find type definitions for this import for some reason. Could you look into that and see if you can resolve it please?

this issue has been resolved in my last commit